gobblin-runtime/src/test/java/gobblin/runtime/mapreduce/MRJobLauncherTest.java

```diff
@@ -118,7 +128,34 @@ public void testLaunchJobWithFork()
     jobProps.setProperty(ConfigurationKeys.WRITER_OUTPUT_DIR + ".1", "gobblin-test/basicTest/tmp/taskOutput");
     jobProps.setProperty(ConfigurationKeys.DATA_PUBLISHER_FINAL_DIR + ".0", "gobblin-test/jobOutput");
     jobProps.setProperty(ConfigurationKeys.DATA_PUBLISHER_FINAL_DIR + ".1", "gobblin-test/jobOutput");
-    runTestWithFork(jobProps);
+    this.jobLauncherTestHelper.runTestWithFork(jobProps);
+  }
+
+  /**
+   * Byteman test that ensures the {@link MRJobLauncher} successfully cleans up all staging data even when an exception
+   * is thrown in the {@link MRJobLauncher} collectOutput method.
+   */
+  @Test()
+  @BMRule(name = "testJobCleanupOnError", targetClass = "gobblin.runtime.mapreduce.MRJobLauncher",
```

Comment:
Can you put each annotation variable in a separate line so it's easier to see the values of them?